src/mpi/romio/adio/ad_daos/ad_daos_open.c

```diff
@@ -589,37 +300,44 @@ void ADIOI_DAOS_Open(ADIO_File fd, int *error_code)
 {
     struct ADIO_DAOS_cont *cont = fd->fs_ptr;
     static char myname[] = "ADIOI_DAOS_OPEN";
+    char *dfs_dir = NULL;
```

Comment:
My test build says this `dfs_dir` is an unused variable. Otherwise the rest of this looks fine.

Reply:
@mchaarawi if you remove that, and your signed-off-by line in the commit message, we should be able to merge this. Please also rebase to the latest mpich/main branch. Normally I would do this, but I don't seem to have permission to modify the PR branch in this case.

Reply:
@raffenet thanks for the review. indeed this was a variable that i removed locally but forgot to force push the commit after.
anyway should be good now and branch rebased with main. I actually just realized the main "main" branch name  ;-)
That must have broken many scripts I imagine!